### DISCUSSION TITLE
Visibility for "Internal" Github package

### DISCUSSION THREAD

**Discussion by sydseter** [community/community#40536] [Enterprise]
### Select Topic Area

Question

### Body

Hi.

On https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#container-creation-visibility-for-organization-members 

For enterprise customers it says that packages that are created with visibility "internal" are visible to all enterprise users, but when I change the visibility from private to internal the UI only say that the package will be visible to all members of a specific organization (In our case "AdmincontrolAS"). Is this just a inconsistency in the UI? Is what actually is meant here that the package is visible to the entire enterprise?
![change visibility2](https://user-images.githubusercontent.com/1244005/205030789-78fbbc72-ebd5-4c54-bf71-e04034392a65.PNG)

**Comment by ossanna16** [comment]
Hi there @sydseter and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @sydseter, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by Timmmy-nlb** [comment]
Also, please make it possible to set the default "Internal" globally - as a enterprise policy.
After all, as an enterprise user/company, you want the settings to be consistent across all organizations.
So you have to activate the option "Internal"& "Inherit" manually for every new org.
